samples/dashboards/demo/cloud-monitoring/demo.js

```diff
@@ -0,0 +1,724 @@
+                },
+                tooltip: {
+                    valueDecimals: 0,
+                    valueSuffix: ' bytes'
+                },
+                accessibility: {
+                    description: `The chart is displaying amount of in and out
+                                network operations`,
+                    point: {
+                        valueDescriptionFormat: 'bytes'
+                    }
+                }
+            }
+        },
+        {
+            cell: 'disk-opt',
+            type: 'Highcharts',
+            title: 'Disk operations',
+            connector: {
+                id: 'charts'
+            },
+            columnAssignment: {
+                timestamp: 'x',
+                writeOpt: 'y',
+                readOpt: 'y'
+            },
+            sync: {
+                highlight: true
+            },
+            chartOptions: {
+                chart: {
+                    type: 'column'
+                },
+                xAxis: {
+                    type: 'datetime',
+                    accessibility: {
+                        description: 'Days'
+                    }
+                },
+                tooltip: {
+                    valueDecimals: 0,
+                    valueSuffix: ' operations'
+                },
+                yAxis: {
+                    title: {
+                        text: 'Operations'
+                    },
+                    accessibility: {
+                        description: 'Operations'
+                    }
+                },
+                accessibility: {
+                    description: `The chart is displaying amount of in and out
+                                operations on disk`,
+                    point: {
+                        valueDescriptionFormat: 'operations'
+                    }
+                }
+            }
+        },
+        {
+            cell: 'instances-table',
+            type: 'DataGrid',
+            title: 'Instances',
+            visibleColumns: ['InstanceId', 'InstanceType', 'PublicIpAddress', 'State', 'HealthIndicator'],
+            dataGridOptions: {
+                editable: false,
+                columns: {
+                    InstanceId: {
+                        headerFormat: 'ID'
+                    },
+                    InstanceType: {
+                        headerFormat: 'Type'
+                    },
+                    PublicIpAddress: {
+                        headerFormat: 'Public IP'
+                    },
+                    HealthIndicator: {
+                        headerFormat: 'Health'
+                    }
+
+                },
+                events: {
+                    row: {
+                        click: async function (e) {
+                            if (pollingCheckbox.checked) {
+                                // stop polling when is enabled
+                                await pollingCheckbox.click();
+                            }
```

Comment:
Also rephrasing Stian. It feels like a bug that the polling stops when I switch instances. Can we keep it running? If that causes problems, maybe just a temporary stop then restart after `setupDashboard` has run?